spec/models/hydra/works/collection_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe Hydra::Works::Collection do
+  
+  describe "#save" do
+    it 'is a Hydra::PCDM::Collection' do 
```

Comment:
I'm not sure why this is important.  I'd say it is important for col.resource.type to include http://pcdm.org/ns/Collection